test/board_test.rb

```diff
@@ -11,37 +11,38 @@ def test_board_exists
   end
 
   def test_board_has_16_cells
-    skip
+    #skip
     board = Board.new
-    assert_equal 16, board.count
+    assert_equal 16, board.cells.size
+    #count is an array method, size is for hashes
```

Comment:
I was testing this in pry & .count was working on hashes that I created, returning the number of key/value pairs.

Reply:
I'd say give it a shot. if it works, we keep it!

Reply:
Ah, if we use `board.cells.count` it'll output the same thing. You're right!